handlers/update_resource.go

```diff
@@ -0,0 +1,57 @@
+package handlers
+
+import (
+	"log"
+
+	"github.com/go-openapi/runtime/middleware"
+	"github.com/sul-dlss-labs/taco"
+	"github.com/sul-dlss-labs/taco/generated/models"
+	"github.com/sul-dlss-labs/taco/generated/restapi/operations"
+	"github.com/sul-dlss-labs/taco/persistence"
+)
+
+// NewDepositResource -- Accepts requests to create resource and pushes them to Kinesis.
+func NewUpdateResource(rt *taco.Runtime) operations.UpdateResourceHandler {
+	return &updateResourceEntry{rt: rt, repository: rt.Repository()}
+}
+
+type updateResourceEntry struct {
+	rt         *taco.Runtime
+	repository persistence.Repository
```

Comment:
since repository is a property of `taco.Runtime` is there a reason to duplicate it?

Reply:
Was following the convention in the other handlers, maybe not and will test.